File: python/tutorial/15-calledid-branching.py
# --------------------------------------------
# changing behavior based on number called
# See http://www.tropo.com for more info
# --------------------------------------------

answer()

if (currentCall.calledID == '4075551111'):
    say("Hello Andrew")
if (currentCall.calledID == '4075552222'):
    say("Hello Brian")

hangup()

File: ruby/tutorial/comfy/15-calledid-branching.rb
# -----------
# Changing behavior based on number called
# -----------
call = Tropo::TropoCall.new($incomingCall)
call.answer

if $currentCall.callerID == '4075551111'
  call.say 'Hello Andrew.'
elsif $currentCall.callerID == '4075552222'
  call.say 'Hello Brian.'
end

call.hangup